app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java

```diff
@@ -1,5 +1,36 @@
 package com.example.calculadoramvp.mvp.view;
 
-public class CalculatorView {
+import android.app.Activity;
+import android.widget.TextView;
+import com.example.calculadoramvp.R;
 
-}
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+
+
+public class CalculatorView extends ActivityView {
+
+    @BindView(R.id.visor_operation) TextView visorOperation;
+    @BindView(R.id.visor_result) TextView visorResult;
+
+    public CalculatorView(Activity activity){
+        super(activity);
+        ButterKnife.bind(this,activity);
+
+        //TODO
```

Comment:
Please add some message into the TODO comment to understand the behavior.

Reply:
done